### DISCUSSION TITLE
I cant upload code with usb on my blue pill

### DISCUSSION THREAD

**Discussion by rupsimrd** [stm32duino/Arduino_Core_STM32#1505] [Q&A] [answered]
hello everyone I cant upload to my blue pill with usb bootloader. arduino ide says that acces is denied. when i open serial port, i see:

Congratulations, you have installed the STM32duino bootloader
See https://github.com/rogerclarkmelbourne/STM32duino-bootloader

For more information about Arduino on STM32
See https://www.stm32duino.com/

here are my settings:
![image](https://user-images.githubusercontent.com/87750161/134481045-0d1a8fd7-9f24-42c2-baa0-f1ff9da87b9e.png)

 what should i look for?

**Comment by fpistm** [comment, marked as answer]
Hi @rupsimrd 
You do not use this STM32 core but this one:
https://github.com/rogerclarkmelbourne/Arduino_STM32

**Reply by rupsimrd** [reply]
just tried it, that doesnt help it

**Reply by fpistm** [reply]
What did you tried?

**Reply by rupsimrd** [reply]
i changed everything according to https://github.com/rogerclarkmelbourne/Arduino_STM32/wiki/Installation

**Reply by fpistm** [reply]
OK but as I said you used an other core than Arduino_Core_STM32 so please use the forum or open an issue on the correct repo.
